### DISCUSSION TITLE
Jellyfin-Roku not available in my region

### DISCUSSION THREAD

**Discussion by GregWoods** [jellyfin-archive/jellyfin-roku-legacy#1420] [Q&A] [answered]
I am in the UK, and trying to download the Jellyfin-roku app from the provided link just brings me to the apps home page https://channelstore.roku.com/en-gb/browse
Searching for Jellyfin also fails,
Is there a workaround for this?

**Comment by TheBetelgeuze** [comment, marked as answer]
There is an issue with Jellyfin on Roku being changed to a beta channel for some reason. You should be able to install it by going to the link below, it will just warn you it is a beta app, make sure you are signed in before otherwise it takes you to another page.

https://my.roku.com/account/add?channel=jellyfin

**Reply by GregWoods** [reply]
Thank you,  that worked.

**Reply by cewert** [reply]
I made a ticket to track this issue: https://github.com/jellyfin/jellyfin-roku/issues/1424
